homeassistant/helpers/template.py

```diff
@@ -884,6 +884,16 @@ def ordinal(value):
     )
 
 
+def from_json(value):
+    """Convert a string to a JSON object."""
```

Comment:
this description is incorrect?

Reply:
I believe it's correct. The filters are opposites. One serializes an object to a JSON string and the other parses a JSON string back into an object.

Reply:
I've changed this to, "Convert a JSON string to an object."